### ISSUE TITLE
End Cake

### ISSUE THREAD

**Issue by fxceptioN** [Darkosto/SkyFactory4Issues#864]
I created an end cake and went into "The End". I put the Overworld Cake in the very middle obsidian block. I managed to die in the end. went back in got my corpse, went back up to come back to Overworld. Overworld Cake is not there. I killed myself. made an Overworld Cake, Placed it in the same location previously, and it disappeared again after dying and going back to get my corpse. Not sure what's going on here or if this in fact meant to happen. Couldn't figure it out.

**Comment by Mrbysco** [comment]
Have you tried rejoining the world

**Comment by fxceptioN** [comment]
Yes I have, it's not there. Nor the one's i made and put in the same place. I did remember the end has a portal back, but still would like to know if this is supposed to happen or if it's a bug.

**Comment by Mrbysco** [comment]
Did you keep any of the other stuff you made? Or did your world get frozen

**Comment by fxceptioN** [comment]
What do you mean?

**Comment by Mrbysco** [comment]
When the lost your cake did you lose anything else or just the cake.

**Comment by Durandle** [comment]
it seems to be an issue with spawning into the end and it destroys ove world cakes that are on the obsidian platform i just decided i may as well jump through the vanilla end portal to get back

**Comment by trsblur** [comment]
The spawn platforms in the end and the nether both seem to delete anything build on top of them when teleporting into the dimensions

**Comment by scollenburg** [comment]
Possibly related: https://github.com/DarkPacks/SkyFactory-4/issues/1181

**Comment by progheal** [comment]
Vanilla End platform do delete three blocks above the obsidian platform. Quoting https://minecraft.gamepedia.com/End :
> Upon arrival in the End, the player will spawn on a 5×5×1 obsidian platform with 3 air blocks above it. 
> ...
> The obsidian platform is regenerated each time a player enters the End, meaning that any blocks or block entities placed in the 5×5×3 space above the platform will be destroyed and replaced with air blocks.

Don't know if this is the case if you arrive the End by cake and whether it affects other dimensional cakes though.
